UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/HealthSystemBase.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+
+namespace HealthV2.Living.PolymorphicSystems
+{
+	[System.Serializable]
+	public abstract class HealthSystemBase
+	{
+
+		//TODO Out of order adding systems  or Ensure fixed order?
+
+
+		[HideInInspector] public LivingHealthMasterBase Base;
+
+		public virtual void InIt()
```

Comment:
can we get some docs on these abstract functions?

Reply:
I concur, we need the methods to have the xml documentation so we know how to use them. What's the difference between ``Init`` and ``StartFresh``?